packages/insured-bridge-relayer/src/Relayer.ts

```diff
@@ -891,14 +899,14 @@ export class Relayer {
           };
           _deposit.depositHash = this.l2Client.generateDepositHash(_deposit);
           if (_deposit.depositHash === relay.depositHash) {
+            deposit = _deposit;
```

Comment:
minor bug fix I found: `deposit` will always be `undefined` if we don't place this log after resetting `deposit = _deposit`